src/main/java/com/gimp/GimPluginConfig.java

```diff
@@ -65,6 +65,25 @@ default int showFootsteps()
 		return 0;
 	}
 
+	@ConfigItem(
+		keyName = "pings",
+		name = "Pings",
+		description = "Enables party pings (shift + left-click)",
+		position = 3
```

Comment:
If you're adding options 3 & 4 here, make sure the Developer ConfigSection and the showSelf ConfigItem are also shifted, to 5 & 6.